### PR TITLE
Vet specialties

### PR DESCRIPTION
**Pull request by jonathanjenin**
Do not merge!

### CODE REVIEW

**Comment by jonathanjenin** [comment]
Can be merged. :)

File: db/seeds.rb [hunk 1]
```diff
@@ -5,11 +5,31 @@
...
+
+  specialties.each do |s|
+    specialty = Specialty.new
+    specialty.name = s
+    specialty.save!
+    p "Saved #{s} as a vet' specialty."
+  end
+end
+
 vet_1 = User.create!({
```
Comment (randzoh) [inline db/seeds.rb:33]: vet_specialty should be modified

**Comment by jonathanjenin** [comment]
Fixed the Dr thing too. :)

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -6,7 +6,15 @@ def my_profile; end
...
-    @user.update(user_params)
+    # Cleaning empty specialty
+    specialty_ids = params[:user][:specialty_ids].select {|s| !s.empty? }
+
+    # Updating user specialties associations
+    @user.specialties = specialty_ids.map { |id| Specialty.find(id) }
+
+    # Saving user
+    @user.update!(user_params)
+    raise
```
Comment (randzoh) [inline app/controllers/users_controller.rb:17]: forgotten raise?

Comment (jonathanjenin) [reply app/controllers/users_controller.rb]: Very good catch, thank you!

**Pull request merged**